doc/nrf/installation/recommended_versions.rst

```diff
@@ -332,12 +332,19 @@ They can all be installed using the ``doc/requirements.txt`` file using ``pip``.
 nRF Command Line Tools
 **********************
 
-`nRF Command Line Tools`_ is a package of tools used for development, programming, and debugging of Nordic Semiconductor's nRF51, nRF52, nRF53 and nRF91 Series devices.
-Among others, this package includes the following prerequisites for the |NCS|:
+`nRF Command Line Tools`_ is a package of tools used for development, programming, and debugging of Nordic Semiconductor's nRF51, nRF52, nRF53, nRF54H, and nRF91 Series devices.
```

Comment:
Not related to my initial feedback, but is nRF54L not in this list on purpose?

Reply:
It is not there on purpose, as only nRF54H docs mention the nRF CLT are required in their getting started guide.

Reply:
Then someone should add it, trying to do the `west flash` as described in the nRF54L guide without CLT installed causes this:

```
FATAL ERROR: required program nrfjprog not found; install it or add its location to PATH
```

Reply:
@annwoj FYI